model/portableElement/PortableElementService.php

```diff
@@ -38,6 +38,9 @@ class PortableElementService implements ServiceLocatorAwareInterface
 {
     use ServiceLocatorAwareTrait;
 
+    const PORTABLE_CLASS_INTERACTION = 'oat\\taoQtiItem\\model\\qti\\interaction\\CustomInteraction';
```

Comment:
Shouldn't you use the static member `::class` on the real class, once imported  ? 
```php
use oat\taoQtiItem\model\qti\interaction\CustomInteraction;
//...
CustomInteraction::class
```

(Needs validation from a backend dev)

Reply:
you right, it is prettier this way -> done